cfme/web_ui/search.py

```diff
@@ -16,10 +16,8 @@
         search_field=Input("search_text", "search[text]"),
 
         # The icon buttons for searching
-        search_icon={
-            "5.4":
-            "//div[@id='searchbox']//div[contains(@class, 'form-group')]"
-            "/*[self::a or (self::button and @type='submit')]"},
+        search_icon=("//div[@id='searchbox']//div[contains(@class, 'form-group')]"
```

Comment:
**Optional**
I've got this one covered in #4180 with updated xpath if you want to leave it off here. No worries either way but you might have to merge my changes if that PR gets in first.